server/apps/main/tests/test_views.py

```diff
@@ -304,6 +304,127 @@ def test_share_exp_submit_new_experience(self):
         # Check that there is a redirect after
         assert response.status_code == 200
 
+    @vcr.use_cassette(
+        "server/apps/main/tests/fixtures/share_experience.yaml",
+        record_mode="none",
+        filter_query_parameters=["access_token"],
+        match_on=["path"],
+    )
+    def test_share_exp_redirect_confirmation(self):
+        """
+        Test that the user is redirected to the user profile
+        given the profile is incomplete and the experience will be
+        shared publicly and for research
```

Comment:
Based on the function title (& logic below) I guess that this one actually tests that the redirect is to the confirmation page?